app/helpers/admin/filter_helper.rb

```diff
@@ -25,7 +25,7 @@ def filter_link_to(text, link_to_params, link_class_params = link_to_params)
   end
 
   def table_link_to(icon, text, path, **options)
-    link_to safe_join([material_symbol(icon), text]), path, options.merge(class: 'table-action-link')
+    link_to safe_join([material_symbol(icon), text]), path, options.merge(class: safe_join(['table-action-link', options[:class]], ' '))
```

Comment:
This didn't previously seem to support adding additional class names to the links.